test/components/binding-redis.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
Can't we reuse the existing MQTT binding?

Reply:
Let's add it for Redis specifically as well maybe as it's a common component?

Reply:
Sure, shall we remove MQTT then too? It will ease setup. Redis is more commonly available.

Reply:
I would keep MQTT as we want to test on different platforms, no? MQTT is used quite broadly btw :)

Reply:
To me it seems fine to have redis and mqtt both.
I used redis here specifically, as for redis metadata "key" is crucial (https://github.com/dapr/components-contrib/blob/b18e73d028e1b6553973efd449085a88d0bb9690/bindings/redis/redis.go#L73) or else this output binding won't serve any purpose.

Reply:
Ok, I would suggest having one working e2e test (from any component for what it's worth), and then uncomment the other one later.